.github/workflows/avm.platform.assign-issue.yml

```diff
@@ -0,0 +1,38 @@
+# Workflow for notifying and assigning issues on creation
+name: Issue Creation - notify and assign team
```

Comment:
I guess this should have the same name as the file to follow the pattern and order.

Reply:
done

Reply:
hm... no :D

Reply:
@AlexanderSehr  Please check now. Missed one additional line. Sorry.

Reply:
@rahalan no :D Again, for the pipeline name the ask would be to use the file name as we did for all other pipelines. In your case it would be `name: avm.platform.assign-issue`

Reply:
Changed it to the new file name: avm.platform.set-avm-github-issue-owner-config